packages/catalog-model/schema/kinds/API.v1alpha1.schema.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Maybe yaml would be more comfortable for the schema files (and allows to place comments)

Reply:
Mm maybe. I'll check if ajv-cli supports it

Reply:
It does, I'm using it in other projects

Reply:
Both as input and schema